internal/provider/string.go

```diff
@@ -29,10 +31,12 @@ func stringSchemaV1(sensitive bool) map[string]*schema.Schema {
 		},
 
 		"length": {
-			Description: "The length of the string desired.",
-			Type:        schema.TypeInt,
-			Required:    true,
-			ForceNew:    true,
+			Description: "The length of the string desired. The minimum value for length is 1 and, length " +
+				"must also be >= (min_upper + min_lower + min_numeric + min_special).",
```

Comment:
Nit: it would look nicer in the final doc, if all the resource arguments, are in `code` format.

Reply:
Excellent point. Have fixed.